Expand troubleshooting section of DEVELOPMENT.md

<!--
Thanks for submitting this documentation request. Please fill the template
below, otherwise we will not be able to process this request.
-->

#### Summary
<!-- Summarize the request in a few sentences: -->

I was unable to run a local development stack, blank white screen with missing module error in chrome debugger:

`ERROR in ./node_modules/redux-logic/node_modules/rxjs/operators/index.js
Module not found: Error: Can't resolve '../internal/operators/audit' in '/Users/ben/lorawan-stack/node_modules/redux-logic/node_modules/rxjs/operators'`

@adriansmares  helped me fix it by cleaning the yarn cache
`yarn cache clear`

And then clearing and reinstalling deps
`./mage js:cleanDeps`
`./mage js:deps`

ALSO, suggesting to add solutions for
`Error occured while trying to proxy to: localhost:8080/console/`
and
`Cannot GET /`

#### Why do we need this ?
<!-- Please explain the motivation, for whom, etc. -->

It is really unclear where this error comes from. I re cloned the stack several times, unlinked and linked the js sdk, and tried everything else from the troubleshooting section.

#### What is already there? What do you see now?
<!--
Please add any relevant documentation articles and resources. Screenshots if necessary.
-->

This error is not covered

#### What is missing? What do you want to see?
<!-- Please add documentation sources (forum links, outside sources...), mock-ups if applicable -->

Section addressing this error and suggesting cache clean

#### How do you propose to document this?
<!-- Please think about how this could be fixed. -->

I will add it to DEVELOPMENT.md

#### Can you do this yourself and submit a Pull Request?
<!-- You can also @mention experts if you need help with this. -->

Yes

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Troubleshooting section of `DEVELOPMENT.md`
+
 ### Deprecated
 
 ### Removed

--- a/DEVELOPMENT.md
+++ b/DEVELOPMENT.md
@@ -844,6 +844,13 @@ console.4e67a17c1ce5a74f3f50.js:104 Uncaught TypeError: m.subscribe is not a fun
     at o (console.4e67a17c1ce5a74f3f50.js:1)
     at Object.0 (console.4e67a17c1ce5a74f3f50.js:104)
 ```
+
+or
+
+```
+ERROR in ./node_modules/redux-logic/node_modules/rxjs/operators/index.js Module not found: Error: Can't resolve '../internal/operators/audit' in '/lorawan-stack/node_modules/redux-logic/node_modules/rxjs/operators'
+```
+
 #### Possible causes
 
 ##### Bundle using old JS SDK
@@ -858,6 +865,14 @@ The bundle integrates an old version of the JS SDK. This is likely a caching/lin
     - If you have cloned multiple `lorawan-stack` forks in different locations, `yarn link` might associate the JS SDK module with the SDK on another ttn repository
   - Rebuild the Console and (only after the build has finished) restart The Things Stack
 
+##### Broken yarn or npm cache
+
+##### Possible solutions
+
+- Clear yarn cache: `yarn cache clear`
+- Clear npm cache: `npm cache clear`
+- Clean and reinstall dependencies: `./mage js:cleanDeps js:deps`
+
 #### Problem: The build crashes without showing any helpful error message
 
 #### Cause: Not running mage in verbose mode
@@ -867,6 +882,27 @@ The bundle integrates an old version of the JS SDK. This is likely a caching/lin
 #### Solution
 
 Run mage in verbose mode: `./mage -v {target}`
+
+#### Problem: Browser displays error: `Cannot GET /`
+
+##### Cause: No endpoint is exposed at root
+
+##### Solution:
+
+Console is typically exposed at `http://localhost:8080/console`,
+API at `http://localhost:8080/console`,
+OAuth at `http://localhost:8080/oauth`,
+etc
+
+#### Problem: Browser displays error: `Error occurred while trying to proxy to: localhost:8080/console`
+
+##### Cause: Stack is not available or not running
+
+For development, remember to run the stack with `go run`:
+
+```bash
+$ go run ./cmd/ttn-lw-stack start
+```
 
 #### General advice
 

